src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const sourceObject = sourceArray.reduce((acc, curr) => {
+    const entry = curr.split(':').map((item) => item.trim());
+
+    acc[entry[0]] = entry[1];
+
+    return acc;
```

Comment:
Consider checking if 'entry[0]' is not empty before adding it to the accumulator object to avoid adding undefined keys in case of empty style declarations.